web/src/beta/features/Editor/useLayers.ts

```diff
@@ -18,7 +18,8 @@ export type LayerAddProps = {
 
 export type LayerNameUpdateProps = {
   layerId: string;
-  name: string;
+  name?: string;
+  visible?: boolean;
```

Comment:
This type is `LayerNameUpdateProps`. If you want to make this a general layer update prop and callback please update the naming 🙏🏻